tla/ccfraft.tla

```diff
@@ -163,6 +163,9 @@ Messages ==
     \* spec does this.
     messages
 
+MessagesOf(s) ==
```

Comment:
"Of" feels unnecessarily vague, since these messages are all directional. Can we call this `MessagesTo` (distinct from a possible `MessagesFrom`)?

Reply:
Can s be `source` or at least `src` too?

Reply:
Corrected to `MessagesTo(dest)`